.storybook/main.js

```diff
@@ -72,10 +80,20 @@ export default {
 
 		return mergeConfig(config, {
 			publicDir: "./assets",
+			// Add dependencies to pre-optimization
+			optimizeDeps: {
+				include: [
+					"@whitespace/storybook-addon-html",
+					"@storybook/blocks",
+					"@storybook/theming",
+					"@storybook/components",
+				],
+			},
 			build: {
 				sourcemap: configType === "DEVELOPMENT",
 				manifest: true,
 				minify: configType === "PRODUCTION",
+				cssCodeSplit: false,
```

Comment:
What is this change and is there any way to validate?

Reply:
I can write up a test case, it's an optimization suggestion I thought would reduce the amount of fetches being done on load.
https://vite.dev/config/build-options#build-csscodesplit

Reply:
When writing out the test case, I didn't see any differences in the Network tab for how many files were loaded per component so I removed this update until we find we need it.